packages/FlexGrid/Row/Row.md

```diff
@@ -78,36 +90,85 @@ Use flexbox alignment properties to vertically and horizontally align columns.
 
 ### Distribution
 
-Use flexbox distribution properties to control the negative space around columns.
+Use `FlexGrid` distribution properties to control the negative space around columns.
 
 ```jsx { "props": { "className": "docs_full-width-playground docs_flex-grid-coloring" } }
 <FlexGrid>
   <FlexGrid.Row distribute="around">
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Around</Text></Box>
+      <Box vertical={2}>
+        <Text>Around</Text>
+      </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Around</Text></Box>
+      <Box vertical={2}>
+        <Text>Around</Text>
+      </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Around</Text></Box>
+      <Box vertical={2}>
+        <Text>Around</Text>
+      </Box>
     </FlexGrid.Col>
   </FlexGrid.Row>
 
   <FlexGrid.Row distribute="between">
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Between</Text></Box>
+      <Box vertical={2}>
+        <Text>Between</Text>
+      </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Between</Text></Box>
+      <Box vertical={2}>
+        <Text>Between</Text>
+      </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={3}>
-      <Box vertical={2}><Text>Between</Text></Box>
+      <Box vertical={2}>
+        <Text>Between</Text>
+      </Box>
     </FlexGrid.Col>
   </FlexGrid.Row>
 </FlexGrid>
 ```
 
+### Spacing between rows
+
+Use the [Box](#box) component's `between` prop to add spacing between rows.
+
+_Note:_ When adding spacing between rows, please only use `between` spacing values _5_, _6_ or _7_.
+
+```jsx { "props": { "className": "docs_full-width-playground docs_flex-grid-coloring" } }
+<FlexGrid>
+  <Box between={5}>
```

Comment:
This is improper use. Let's revert it to have `<Box>` nested inside each individual column until we align on a proper solution.